docs/manual/installation.rst

```diff
@@ -45,24 +45,17 @@ To build QXelarator yourself from source and add it to your local Python package
 
 You will need to have SWIG installed for the above to work.
 
-Building the C++ executable from source
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Building the C++ executable from source with Conan
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 This is particularly useful for debugging purposes, as the executable can then be run under ``gdb``, for instance. To do so,
-do not forget to build it in debug mode, otherwise compiler optimizations will make debugging more difficult.
+do not forget to build in debug mode.
 
 .. code-block:: bash
 
-    mkdir build
+    python3 -m pip install --upgrade pip conan
 
-    cd build
+    conan build . -pr=conan/profiles/tests-Debug -b missing
```

Comment:
It should also be stated that `conan profile detect` has to be performed once before the first `conan build`.